pkg/network/dns/cache.go

```diff
@@ -85,14 +89,14 @@ func (c *reverseDNSCache) Add(translation *translation) bool {
 				log.Warnf("%s mapped to too many domains, DNS information will be dropped (this will be logged the first 10 times, and then at most every 10 minutes)", addr)
 			}
 		} else {
-			c.added.Inc()
+			cacheTelemetry.added.Inc()
 			// flag as in use, so mapping survives until next time connections are queried, in case TTL is shorter
 			c.data[addr] = &dnsCacheVal{names: map[Hostname]time.Time{translation.dns: deadline}, inUse: true}
 		}
 	}
 
 	// Update cache length for telemetry purposes
-	c.length.Store(int64(len(c.data)))
+	cacheTelemetry.length.Set(int64((len(c.data))))
```

Comment:
nit: extra parentheses
```suggestion
	cacheTelemetry.length.Set(int64(len(c.data)))
```